rules/windows/persistence_suspicious_image_load_scheduled_task_ms_office.toml

```diff
@@ -19,9 +44,70 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
 language = "eql"
 license = "Elastic License v2"
 name = "Suspicious Image Load (taskschd.dll) from MS Office"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Suspicious Image Load (taskschd.dll) from MS Office
+
+Microsoft Office is a suite of applications designed to help with productivity and completing common tasks on a computer. You can create and edit documents containing text and images, work with data in spreadsheets and databases, and create presentations and posters. As it is some of the most-used software across companies, MS Office is frequently targeted for initial access. It also has a wide variety of capabilities that attackers can take advantage of.
```

Comment:
Might make more sense here to write why MS Office is capable of providing initial access to attackers, rather than explaining what MS Office exactly is IMO.